frontend/src/components/Showtimes/Showtimes.component.vue

```diff
@@ -0,0 +1,347 @@
+              <v-card-actions>
+                <v-btn @click="remove(show.id)">yes</v-btn>
+                <v-spacer/>
+                <v-btn @click="confirmDelete = false">no</v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
+        </v-list-tile>
+      </v-list>
+    </v-flex>
+  </v-layout>
+</template>
+
+<script>
+import TheaterController from 'Controllers/system-admin.controller';
+import ShowtimeController from 'Controllers/showtimes.controller';
+import MovieController from 'Controllers/movies.controller';
+import { Movie } from 'Models/movie.model';
+import { Showtime } from 'Models/showtime.model';
+import { mapGetters } from 'vuex';
+
+export default {
+  data: () => ({
+    valid: true,
+    loading: false,
+    search: '',
+    movies: [],
+    movie: new Movie(),
+    confirmDelete: false,
+    deletingShowtime: 0,
+    editingShowtime: new Showtime(),
+    theaterId: 0,
+    createDialogBool: false,
+    showtime: new Showtime(),
+    modal: false,
+    time: null,
+    showtimes: [],
+    showShowtimes: false,
+    movieName: '',
+    editDialog: false
+  }),
+  computed: {
+    filteredMovies() {
+      if (!_.isEmpty(this.search)) {
+        return this.movies.filter((i) => {
+          return i.title.toLowerCase().includes(_.trim(this.search.toLowerCase()));
+        });
+      }
+      return this.movies;
+    },
+    ...mapGetters([
+      'activeUser'
+    ])
+  },
+  mounted() {
+    this.getTheaterAndMovies();
+  },
+  methods: {
+    createDialog(movie) {
+      this.createDialogBool = true;
+      this.showtime.movie = movie.id;
+    },
+    create() {
+      this.showtime.time = this.time;
+      this.createDialogBool = false;
+      ShowtimeController.create(this.showtime)
+        .then((response) => {
+          this.$alert.success('Successfully added!');
+        })
+        .catch((response) => {
+          this.$alert.error('Error occurred.');
+        });
+    },
+    opetDialog() {
+      this.showDialog = true;
+    },
+    clear() {
+      this.showtime = new Showtime();
+    },
+    getTheaterAndMovies() {
+      this.loading = true;
+      TheaterController.getTheater(this.activeUser.id)
+        .then((response) => {
+          this.theaterId = response.data.id;
+          this.getMovies();
+          this.loading = false;
+        })
+        .catch((response) => {
+          this.loading = false;
+          this.$alert.error('Error occurred.');
+        });
+    },
+    getMovies(id = this.theaterId) {
+      this.loading = true;
+      TheaterController.getMovies(id)
+        .then((response) => {
+          this.movies = response.data;
+          this.loading = false;
+        })
+        .catch((response) => {
+          this.loading = false;
+          this.$alert.error('Error occurred.');
+        });
+    },
+    show(movieId) {
+      MovieController.getShowtimes(movieId)
+        .then((response) => {
+          this.showtimes = response.data;
```

Comment:
nece nam trebati model za showtime?

Reply:
a pa imamo model, sad ga vidim dole. onda bi u `this.showtimes` trebalo da budu modeli
`this.showtimes = _.map(response.data, (showtimeData) => { return new Showtime(showtimeData); })`
ili krace
`this.showtimes = _.map(response.data, x => new Showtime(x))`